### ISSUE TITLE
[T17][Accessibility][Functional][Past Medical] Onset:Date is not read completely

### ISSUE THREAD

**Issue by nvnlatur9** [Mohanme13/Feb-19-Bug-bash#26]
Onset:Date is not read completely on the Past Medical tab -> +Add.
Reading only "Onset:" and missing out "Date".
